### ISSUE TITLE
⛔ 🐞 Bug Report: Images with default widget width in containers overlap each other on mobile

### ISSUE THREAD

**Issue by jamieburchell** [elementor/elementor#22281]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

On the mobile breadkpoint, images with a width of 100% and a widget width of "Default" overlap each other when they are added to a container with flex direction column (or not set).

### Steps to reproduce

1) Add a new container
2) Add two images. Set their width to 100% but leave "Advanced" > "Width" as "Default"
3) View page on mobile breakpoint, images overlap/crop and overflow their container

Setting the widget width to "Full width" works around the issue.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.5.16
	PHP Version: 8.0.28
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 50M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: There are some writing permissions issues with the following directories/files:
		 - WordPress root directory
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: https://www.example.com/wp
	Home URL: https://www.example.com
	WP Multisite: No
	Max Upload Size: 40 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /news/%postname%/
	Language: en-GB
	Timezone: Europe/London
	Admin Email: admin@example.com
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36

== Active Plugins ==
	Category Order and Taxonomy Terms Order
		Version: 1.7.5
		Author: Nsp-Code

	Code Snippets
		Version: 3.3.0
		Author: Code Snippets Pro

	Converter for Media
		Version: 5.8.6
		Author: matt plugins

	Dynamic.ooo - Dynamic Content for Elementor
		Version: 2.9.3
		Author: Dynamic.ooo

	Elementor
		Version: 3.12.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.3
		Author: Elementor.com

	Pods - Custom Content Types and Fields
		Version: 2.9.13
		Author: Pods Framework Team

	Post Types Order
		Version: 2.0.5
		Author: Nsp Code

	Rank Math SEO
		Version: 1.0.112
		Author: Rank Math

	W3 Total Cache
		Version: 2.3.1
		Author: BoldGrid

	Wordfence Security
		Version: 7.9.2
		Author: Wordfence


== Must-Use Plugins ==
	Bedrock Autoloader
		Version: 1.0.3
		Author: Roots

	Disable status tests
		Version: 1.0.0
		Author: Roots

	Register Theme Directory
		Version: 1.0.0
		Author: Roots


== Features ==
	Custom Fonts: 1
	Custom Icons: 0

== Integrations ==
	
	recaptcha_v3: Active


== Elementor Experiments ==
	Optimised DOM Output: Active
	Improved asset loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Accessibility improvements: Active
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive by default
	Editor Top Bar: Inactive by default
	Landing pages: Inactive
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Page Transitions: Inactive
	Notes: Inactive
	Loop: Inactive
	Form Submissions: Active by default
	Scroll Snap: Inactive
	Menu: Inactive by default


== Log ==
	
Log: showing 20 of 352023-04-03 15:05:25 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-03 15:05:25 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.1',
)]
2023-04-12 12:12:04 [info] Update DB has been started [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.1',
  'to' => '3.12.2',
)]
2023-04-12 12:12:05 [info] elementor-pro::elementor_pro_updater Started 
2023-04-12 12:12:05 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-12 12:12:05 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-12 12:12:05 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.1',
  'to' => '3.12.2',
)]
2023-04-24 10:23:15 [info] Update DB has been started [array (
  'plugin' => 'Elementor',
  'from' => '3.12.1',
  'to' => '3.12.2',
)]
2023-04-24 10:23:15 [info] elementor::elementor_updater Started 
2023-04-24 10:23:15 [info] Elementor/Upgrades - _on_each_version Start  
2023-04-24 10:23:15 [info] Elementor/Upgrades - _on_each_version Finished 
2023-04-24 10:23:15 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.1',
  'to' => '3.12.2',
)]
2023-04-24 11:18:39 [info] Update DB has been started [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.2',
  'to' => '3.12.3',
)]
2023-04-24 11:18:39 [info] elementor-pro::elementor_pro_updater Started 
2023-04-24 11:18:39 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-24 11:18:39 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-24 11:18:39 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.2',
  'to' => '3.12.3',
)]
2023-05-02 09:47:00 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Dynamic.ooo - Dynamic Content for Elementor',
  'from' => '2.9.2',
  'to' => '2.9.3',
)]
2023-05-02 09:47:12 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Dynamic.ooo - Dynamic Content for Elementor',
  'from' => '2.9.2',
  'to' => '2.9.3',
)]
2023-05-02 09:47:13 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Dynamic.ooo - Dynamic Content for Elementor',
  'from' => '2.9.2',
  'to' => '2.9.3',
)]

PHP: showing 7 of 7PHP: 2023-04-03 15:07:08 [warning X 110][/var/www/web/app/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-03 15:07:30 [warning X 164][/var/www/web/app/plugins/elementor-pro/modules/dynamic-tags/tags/site-logo.php::32] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-26 12:35:19 [warning X 106][/var/www/web/app/plugins/elementor/app/modules/import-export/module.php::695] Attempt to read property "label" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-26 13:46:29 [error X 1][/var/www/web/app/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on bool in /var/www/web/app/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:295
Stack trace:
#0 /var/www/web/app/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions()
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions()
#2 /var/www/web/app/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func()
#3 /var/www/web/wp/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#4 /var/www/web/wp/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#5 /var/www/web/wp/wp-includes/plugin.php(517): WP_Hook->do_action()
#6 /var/www/web/wp/wp-admin/admin-ajax.php(188): do_action()
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-26 15:46:28 [warning X 85][/var/www/web/app/plugins/elementor/core/dynamic-tags/manager.php::73] Array to string conversion [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-26 23:38:59 [warning X 7][/var/www/web/app/plugins/elementor-pro/modules/posts/skins/skin-content-base.php::265] Attempt to read property "ID" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-28 10:10:51 [error X 1][/var/www/web/app/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on bool in /var/www/web/app/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:295
Stack trace:
#0 /var/www/web/app/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /var/www/web/app/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /var/www/web/wp/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /var/www/web/wp/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#5 /var/www/web/wp/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /var/www/web/wp/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 1 of 1JS: 2023-04-26 14:19:51 [error X 2][https://example.com/app/plugins/elementor/assets/js/editor.js?ver=3.12.2:40462:44] Cannot convert undefined or null to object 



== Elementor - Compatibility Tag ==
	
	Dynamic.ooo - Dynamic Content for Elementor: Compatible
	Elementor Pro: Incompatible

== Elementor Pro - Compatibility Tag ==
	
	Dynamic.ooo - Dynamic Content for Elementor: Compatible
				

````

</details>

**Comment by SolunaCG** [comment]
I can confirm this is a problem! A few weeks ago I had the same problem with one of my sites and after checking the site again I see that this is still a problem. The workaround that @jamieburchell mentioned with setting the image width from "Default" to "Full Width" works but it´d be great if this works as default, too. 

<img width="1939" alt="images-not-overlapping-width100" src="https://user-images.githubusercontent.com/114682524/235655073-b7a40a35-98be-4f56-bbcb-3f532653d955.png">
<img width="1932" alt="images-overlapping-mobile" src="https://user-images.githubusercontent.com/114682524/235655091-acea4a6a-cece-4120-ae87-bb95265c81b0.png">

**Comment by nicholaszein** [comment]
Hello @jamieburchell,

Thank you for reaching out! 🙏 

### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
- Testing with the latest Elementor version and couldn't reproduce the issue.
- It seems you have a lot of plugins running on your site, including one Elementor add-on, and some optimization plugins that might be causing your issue.
- The issue #18741 has no correlation to what you are describing here. 


https://user-images.githubusercontent.com/113993460/235686136-c3e7e12f-1c5f-4ce6-bed8-ed7888f68842.mp4


### ✅ What you can do to solve your issue:
- Isolate the problem. Deactivate all plugins except Elementor and test again.

### 👍 For now, that's all we can help with. 

If you think this is still an issue caused by Elementor, please feel free to open a new 🐞 [Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).

### ⚠️ Note that GitHub issues are reserved for Elementor 🐞 Bug Reports only, and should not be used for support.

❔ If you have questions, or need guidance, help, or general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Kind regards
